src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -74,16 +69,72 @@ internal PredictionTransformerBase(IHost host, ModelLoadContext ctx)
             ms.Position = 0;
             var loader = new BinaryLoader(host, new BinaryLoader.Arguments(), ms);
             TrainSchema = loader.Schema;
+        }
+
+        public abstract ISchema GetOutputSchema(ISchema inputSchema);
+
+        /// <summary>
+        /// Transforms the input data.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns></returns>
+        public abstract IDataView Transform(IDataView input);
 
+        protected void SaveModel(ModelSaveContext ctx)
+        {
+            // *** Binary format ***
+            // model: prediction model.
```

Comment:
>model: prediction model. [](start = 15, length = 24)

Technically the model isn't part of this format, since you're not writing it to the stream, you're writing it somewhere else entirely, but that's OK.

Reply:
fixing it == remove the comment?

---
In reply to: [218818528](https://github.com/dotnet/machinelearning/pull/912#discussion_r218818528) [](ancestors = 218818528)